core-plugins/docs/File-batchsink.md

```diff
@@ -0,0 +1,42 @@
+# File Sink
+
+
+Description
+-----------
+Writes to a filesystem in text, avro, or parquet format.
+
+For the text format, each record is written out as delimited text.
+Non-string fields will be converted to strings using their ``toString()`` Java method,
+so fields should be limited to the string, long, int, double, float, and boolean types.
+
+
+Properties
+----------
+**referenceName:** This will be used to uniquely identify this sink for lineage, annotating metadata, etc.
+
+**path:** The directory to write to. For example, ``hdfs://mycluster.net:8020/my/desired/location``.
+
+**suffix:** Time suffix to append to the path for each run of the pipeline. For example,
+``YYYY-MM-dd-HH-mm`` will take the start time of the pipeline run, convert it into
+year-month-day-hour-minute format, and append that to the path to get the final output directory.
+If not specified, no suffix is used.
+
+**delimiter:** Delimiter used to concatenate record fields. Defaults to a comma ','. (Macro-enabled)
+
+**jobProperties:** Advanced feature to specify any additional properties that should be used with the sink,
+specified as a JSON object of string to string. These properties are set on the job at runtime. (Macro-enabled)
+
```

Comment:
there should be format and schema properties too right ?

Reply:
done